src/PKSim.Core/Services/InteractiveSimulationRunner.cs

```diff
@@ -65,9 +77,13 @@ public Task RunSimulation(Simulation simulation, bool selectOutput) =>
 
       private async Task runSimulationAsync(Simulation simulation, bool selectOutput)
       {
+         _shouldRaiseEvents = _simulationRunOptions.RaiseEvents;
          var cts = new CancellationTokenSource();
-         if (!_cancellationTokenSources.TryAdd(simulation, cts)) //this will prevent from running one that is already running
+         var begin = new DateTime();
+
+         if (!_cancellationTokenSources.TryAdd(simulation, cts))
```

Comment:
somehow you removed the comment ? 
Also probalby the beging should be AFTER this line otherwise it's set but never used?